src/custom/components/Header/URLWarning/URLWarningMod.tsx

```diff
@@ -22,9 +31,14 @@ export const StyledClose = styled(X)`
   }
 `
 
-export default function URLWarning({ url }: { url: string }) {
+export default function URLWarning({ url, announcement }: { url: string; announcement?: React.ReactNode }) {
   const toggleURLWarning = useURLWarningToggle()
   const showURLWarning = useURLWarningVisible()
+  const showAnnouncement = !!announcement
```

Comment:
ultra mega nit: you don't need this line here 😂 
i would just save a line and !!anouncement in the if statement but anyways

Reply:
okay, fair enough

Reply:
If it's ok, i will leave it. Makes it slightly more readable and then you don't need either do the `!!announcement` 2 times, or pass to `isActive` a string when it requires a boolean (it will work, I know, but still is weird to pass other type)

Reply:
I was going to comment the same thing :sweat_smile: 
But then I saw it's used in 2 places, so I agreed with Anxo's decision